### PR TITLE
Pass through servo in maker toolkit

### PR DESCRIPTION
**Pull request by bcjordan**
So @mrjoshida can make robots manipulate the world. Enjoy responsibly.

### CODE REVIEW

File: apps/src/makerlab/BoardController.js [hunk 1]
```diff
@@ -116,6 +116,13 @@ export default class BoardController {
       CapTouch: PlaygroundIO.CapTouch,
       Tap: PlaygroundIO.Tap,
       Accelerometer: five.Accelerometer,
+      /**
+       * @link https://en.wikipedia.org/wiki/Three_Laws_of_Robotics
+       * 1. A robot may not injure a human being or, through inaction, allow a human being to come to harm.
+       * 2. A robot must obey orders given it by human beings except where such orders would conflict with the First Law.
+       * 3. A robot must protect its own existence as long as such protection does not conflict with the First or Second Law.
```
Comment (islemaster) [inline apps/src/makerlab/BoardController.js:123]: Beware - you've omitted [the zeroth law](https://en.wikipedia.org/wiki/Three_Laws_of_Robotics#Zeroth_Law_added) :trollface: :robot:.

**Comment by islemaster** [comment]
LGTM.
robots

**Comment by islemaster** [comment]
FYI, Circle failing on `Won't build apps with staged changes` may be what I just encountered re: yarn.lock.

File: apps/src/makerlab/BoardController.js (hunk 1 above)
Comment (bcjordan) [reply apps/src/makerlab/BoardController.js:123]: 😅

**Pull request merged**